Library/Homebrew/cask/lib/hbc/artifact/pkg.rb

```diff
@@ -50,7 +50,13 @@ def run_installer(command: nil, verbose: false, **_options)
         end
         with_choices_file do |choices_path|
           args << "-applyChoiceChangesXML" << choices_path if choices_path
-          command.run!("/usr/sbin/installer", sudo: true, args: args, print_stdout: true)
+          logged_in_user = Utils.current_user
+          env = {
+            "LOGNAME" => logged_in_user,
+            "USER" => logged_in_user,
+            "USERNAME" => logged_in_user,
```

Comment:
> This PR proposes to fix the issue by setting LOGNAME, USER, and USERNAME to the same values as the graphical PKG installer does.

Is `USERNAME` set by this specific installer or by `/usr/sbin/installer` in general?

Reply:
I honestly have no idea. I never bothered to investigate _where_ exactly those three variables are exported in the chain of parent processes of the post-install script.

Rationale: Even though I can confirm that the `dymo-label` installer works fine without `USERNAME`, let’s not assume that any installer will.